docs/content/preview/manage/backup-restore/export-import-data.md

```diff
@@ -40,16 +45,32 @@ To export a single database with all its tables, indexes, and other local artifa
 ./postgres/bin/ysql_dump -d <db-name> > <file>
 ```
 
-* *db-name* is the name of the database to be exported.
-* *file* is the path to the resulting SQL script file.
+- *db-name* is the name of the database to be exported.
+- *file* is the path to the resulting SQL script file.
 
 For example, to export the `mydb` database into a file called `mydb-dump.sql` located in the `backup` folder, the command would be as follows:
 
 ```sh
 ./postgres/bin/ysql_dump -d mydb > backup/mydb-dump.sql
 ```
 
-For more details and a list of all available options, see [ysql_dump reference](../../../admin/ysql-dump/).
+## Export a single table
+
+To export a single database table, use the `ysql_dump` utility by executing the following command:
+
+```sh
+./postgres/bin/ysql_dump -t <table-name> <db-name> > <file>
```

Comment:
Why dont we use ./postgres/bin/ysql_dump -t <table-name> <db-name> -f <file> as specified in the docs: https://docs.yugabyte.com/preview/admin/ysql-dump/#f-file-file-file

Reply:
Not sure. Docs and examples taken from pg_dump docs https://www.postgresql.org/docs/11/app-pgdump.html